Common/src/main/java/jeresources/jei/mob/MobWrapper.java

```diff
@@ -19,18 +26,20 @@
 import net.minecraft.world.entity.animal.Turtle;
 import net.minecraft.world.entity.boss.enderdragon.EnderDragon;
 import net.minecraft.world.entity.boss.wither.WitherBoss;
-import net.minecraft.world.entity.monster.*;
+import net.minecraft.world.entity.monster.AbstractIllager;
+import net.minecraft.world.entity.monster.Blaze;
+import net.minecraft.world.entity.monster.Creeper;
+import net.minecraft.world.entity.monster.EnderMan;
+import net.minecraft.world.entity.monster.Ghast;
+import net.minecraft.world.entity.monster.Husk;
+import net.minecraft.world.entity.monster.Witch;
 import net.minecraft.world.entity.monster.piglin.AbstractPiglin;
 import net.minecraft.world.entity.npc.Villager;
 import net.minecraft.world.entity.npc.WanderingTrader;
 import net.minecraft.world.item.ItemStack;
-import org.jetbrains.annotations.NotNull;
-
-import java.util.Collections;
-import java.util.List;
 
 public class MobWrapper implements IRecipeCategoryExtension, IRecipeSlotTooltipCallback {
-    private final MobEntry mob;
+    public final MobEntry mob;
```

Comment:
This can be private again if you add the 2 methods I proposed `hasSpawnEgg` and `getSpawnEgg`

Reply:
Private again